lib/oxidized/model/supermicrosseg48tg4.rb

```diff
@@ -0,0 +1,45 @@
+# Developed against:
+# #show version
+# Switch ID       Hardware Version                Firmware Version
+# 0               SSE-G48-TG4   (P2-01)           1.0.16-9
+
+class SupermicroSSEG48TG4 < Oxidized::Model
+  prompt (/^(\e\[27m)?[ \r]*\w+# ?$/)
+
+  cfg :ssh do
+    post_login 'no cli pagignation'
+    pre_logout 'exit'
+  end
+
+  cfg :secret do |cfg|
+    cfg.gsub!(/^(snmp community) .*/, '\1 <hidden>')
+    cfg
+  end
+
+  cmd 'show system information' do |cfg|
+    cfg.sub! /^Device Up Time.*\n/, ''
+    cfg.delete! "\r"
+    comment(cfg.each_line.to_a[1..-2].join).gsub(/ +$/, '')
+  end
+
+  cmd 'show running-config' do |cfg|
+    comment_next = 0
+    cfg = cfg.delete("\r").gsub(/ +$/, '').each_line.map {|l|
+      next '' if l.match /^Building configuration/
+
+      if l.match /^Switch ID.*Hardware Version.*Firmware Version/ then
```

Comment:
do you mind posting a sample of the problem you are solving here?

Reply:
The output of `show running-config` starts with these lines:
```
Building configuration...
Switch ID       Hardware Version                Firmware Version
0               SSE-G48-TG4   (P2-01)           1.0.16-9
                                  
ip address [...]
```

`Building configuration...` should be removed, and the `Switch ID[...]` and subsequent line is not actually configuration, and should thus be commented.